packages/components/Button/src/ButtonTokens.ts

```diff
@@ -2,14 +2,18 @@ import { Theme } from '@fluentui-react-native/framework';
 import { globalTokens } from '@fluentui-react-native/theme-tokens';
 import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { ButtonTokens } from './Button.types';
+import { Platform } from 'react-native';
 
 export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
   ({
     block: {
       width: '100%',
     },
     medium: {
-      padding: globalTokens.spacing.sNudge - globalTokens.stroke.width.thin,
+       padding: Platform.select({
+        android: 6,
+        default: globalTokens.spacing.sNudge - globalTokens.stroke.width.thin,
+       }),
```

Comment:
Same comment in the iOS review: can we move this to a platform specific file?

Reply:
This is just a unblocking fix for the tester to load, and we have a issue to track the style updates so that the iOS/ Android styling comes from a different file. At this point, I have not yet fully tracked the set of token changes needed for Button, so leaving this to fix up in the next changes. Reviewed your inputs on Lynn's draft, agree that this is not a pattern for platform styling we would like to be in place.